package.json

```diff
@@ -97,6 +97,7 @@
     "eslint-plugin-node": "^11.1.0",
     "eslint-plugin-promise": "^4.2.1",
     "eslint-plugin-standard": "^4.0.1",
+    "eslint-plugin-typescript-formatter": "0.0.2",
```

Comment:
Despite the amazingly high version number, this is actually the best one I could find.

Reason is that it relies on the compiler instead of trying to reimplement everything (see this for details: https://github.com/typescript-eslint/typescript-eslint/issues/1824)